src/game/client.cpp

```diff
@@ -3240,142 +3240,29 @@ namespace client
         }
     }
 
-    int serverstat(serverinfo *a)
+    bool serverinfocompare(serverinfo const*a, serverinfo const*b)
```

Comment:
New naming convention would suggest `serverinfo_compare`. But I do wonder why this is not implemented as an operator directly?

Reply:
Also, missing spaces.

```suggestion
    bool serverinfocompare(serverinfo const* a, serverinfo const* b)
```